core/src/com/unciv/ui/screens/civilopediascreen/CivilopediaImageGetters.kt

```diff
@@ -26,17 +26,18 @@ internal object CivilopediaImageGetters {
     private fun terrainImage(terrain: Terrain, ruleset: Ruleset, imageSize: Float): Actor {
         val tile = Tile()
         tile.ruleset = ruleset
+        val baseTerrainFromOccursOn = Constants.grassland.takeIf { it in terrain.occursOn && it in ruleset.terrains }
```

Comment:
We can avoid referencing grassland by name entirely, sounds cleaner

Reply:
Out of scope question: but wouldn't a better idea be to define the default here, but add something to `civilopeidaText` to define which image the entry is using? There's plenty of other use cases for it anyways (such as defining a building's image to be an alias of another building or unit directly in the buildings json instead of in the atlas

Reply:
(I know I'm going pretty off topic for this. You can stop me when you want)
> IMO automatic atlas aliasing is perfectly fine.

Counterargument: Rekmod has numerous "Great Work of Whatever" that would all love to share the same icon. Via Atlas aliasing, I kinda only have 1 of 2 options
1. Copy the image over and over and over. That's 33 extra images all matches one of the images already in the mod, which can take up quite a lot of space. And that's the easy example where all I have to do is copy-paste 3 images a bunch of times and just rename stuff. That's not even looking at the many one off cases of a building or promotion explicitly wanting to copy some other image
2. Cheat, and start manually editing the atlas. I don't think anyone really wants to edit the atlas, and this solution has to be reapplied every time you update images
The worst case scenario is [the leader mission mod](https://github.com/Jiangeshi/Leader-Mission-2-Rising-Power) which not only have to deal with the many issues in trying to run a custom atlas, but also runs 80 copies of the same image for all of the leaders

The other side of this is if the was a way to define images or icons or whatever easily in the json, we wouldn't need to define it in code here. It could just be that river's icon is just whatever you want or some combination similar to what's in the tileset jsons or whatever

Reply:
(Funny, you commented just before I was going to add in this)
> And if your third-party tool is more pain than joy

Actually, that's not the only issue. On android, custom atlases are fine enough, but on desktop (especially when I'm testing in Studio), that mod I link actually tends to redo the atlas, which, because the images don't actually match their building/unit/terrain/etc names most of the time, it actually ends up breaking the majority of the images. So the downside of custom atlases isn't just setting it up, but rather making sure it works on all platforms